src/cpu/x64/cpu_isa_traits.hpp

```diff
@@ -60,8 +63,12 @@ enum cpu_isa_bit_t : unsigned {
     amx_bf16_bit = 1u << 11,
     avx_vnni_bit = 1u << 12,
     avx512_vpopcnt_bit = 1u << 13,
+    // Fill in hints from most significant bit to least significant bit
+    prefer_ymm_bit = 1u << (cpu_isa_total_bits - 1),
```

Comment:
The same here (copied from oneDNN 2.3).
I should have provide more context.
Basically we are copying oneDNN 2.3 changes related to mixing vex and nonVEX (especially in case of avx2 alderlake).
To keep the change simple and as close as possible to oneDNN 2.3 implementation I have also backported those cpu_isa_traits updates.
I think we are not really interested in updating / correcting them unless we are sure it gives performance improvement for 2021/4 release.
Maybe @dmitry-gorokhov has some thoughts about this.
Thank you for the comments anyway.

Reply:
@chenhu-wang btw second commit contains additional uni_* updates for XMM related methods (which oneDNN 2.3 does not have) because we noticed that our mkldnn nodes still contains mixing vex and nonVEX instructions.